RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/Encoders.kt

```diff
@@ -9,11 +9,11 @@ import kotlin.math.min
 import kotlin.math.round
 
 class PositionVelocityPair(
-        @JvmField val position: Int, @JvmField val velocity: Int,
-        @JvmField val rawPosition: Int, @JvmField val rawVelocity: Int
+    @JvmField val position: Double, @JvmField val velocity: Double,
+    @JvmField val rawPosition: Double, @JvmField val rawVelocity: Double
  )
 
-sealed interface Encoder {
```

Comment:
Are the odometry sensors modeled as a different kind of encoder? What alternate implementations are you pondering and why shouldn't they be included in this module?

Reply:
Currently in my road-runner-ftc fork I am representing the OTOS position and the Pinpoint raw encoder data as encoder implementations only for tuning. https://github.com/jdhs-ftc/road-runner-ftc-otos/blob/master/RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/OTOS.kt and https://github.com/jdhs-ftc/road-runner-ftc-otos/blob/pinpoint/RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/Pinpoint.kt (note: for the pinpoint I am currently using the raw odometry encoder passthrough for tuning, not the calculated pose, and the comments reflect that)
I mainly did this because it seemed the easiest way to use them for tuning. I have not included them in this PR because honestly I'm not sure they're up to roadrunner standards for quality and testing, but making this no longer sealed would allow me to much more easily develop and/or distribute them.
For actual following I'm using modifications to updatePoseEstimate to set the field centric pose directly, as previously discussed; that does not use these encoder modifications at all.